packages/graphback-cli/src/helpers/db.ts

```diff
@@ -77,7 +78,11 @@ Run ${chalk.cyan(`npm run develop`)} to start the server.
 export const createDB = async (): Promise<void> => {
   const configInstance = new ConfigBuilder();
   checkDirectory(configInstance)
-  await dropDBResources(configInstance)
+
+  if (configInstance.config.db.initialization === DatabaseInitializationStrategy.DropCreate) {
```

Comment:
Dropping resources was intiial idea and also the biggest drawback we had. Since we have very basic migration I think dropping makes no sense anymore in future supported version. It was just quick hack to get us going, but we never actually catched up with doing it properly till today. I have no strong opinion, but do you see any use case where dropping entire database will be an sensible option. 

Dropping databases can have a serious impact if someone missconfigure their project. Once we add migration we will start comming with the cases to make modifications in non-destructive way. Dropping db kinda goes against it in really obvious way.

Reply:
Someone in developer mode might want their database wiped every time. I was modelling it from Entity Framework:

https://www.entityframeworktutorial.net/code-first/database-initialization-strategy-in-code-first.aspx

But as you said, an accidental misconfiguration could be catastrophic.

Should we add a separate command to drop the database? How will someone in developer mode wipe and start clean if we remove it?

Reply:
Actually this is a really good perspective. How to prevent pushing this in production is another idea.